packages/blockchain-link/src/workers/blockbook/websocket.ts

```diff
@@ -52,7 +53,7 @@ export class BlockbookAPI extends BaseWebsocket<BlockbookEvents> {
             agent: this.options.agent,
             headers: {
                 Origin: 'https://node.trezor.io',
-                'User-Agent': 'Trezor Suite',
+                'User-Agent': `Trezor Suite ${getSuiteVersion()}`,
```

Comment:
While I believe this shouldn't break anything and therefore can be merged, I see two possible downsides:

1) As the comment stated, web builds ignore the options (second param) passed to `WebSocket`, therefore this won't work in web Suite. Which is probably expected, right?

2) `getSuiteVersion` util just returns anything what's in `process.env.VERSION`, so if a 3rd party uses Connect in Node environment, it's possible to have something like `Trezor Suite 9.x.x` there, because of connect version. Again, not a big deal?

Reply:
1. is fine
2. I am fine with this